public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -41,368 +41,361 @@ const copyHasTemplate = (test: EpicTest, template: string): boolean =>
       variant.paragraphs.some(para => para.includes(template)),
   );
 
-interface EpicTestEditorProps {
-  test: EpicTest;
-  epicEditorConfig: EpicEditorConfig;
-  onChange: (updatedTest: EpicTest) => void;
-  onValidationChange: (isValid: boolean) => void;
-  visible: boolean;
-  editMode: boolean;
-  onDelete: () => void;
-  onArchive: () => void;
-  onTestSelected: (testName: string) => void;
-  testNames: string[];
-  testNicknames: string[];
-  testNamePrefix?: string;
-  createTest: (newTest: EpicTest) => void;
-}
-
-const EpicTestEditor: React.FC<EpicTestEditorProps> = ({
```

Comment:
Most of this file hasn't changed. It now takes the standard `TestEditorProps` and renders the `StickyTopBar`.